connexion/app.py

```diff
@@ -96,7 +96,7 @@ def common_error_handler(exception):
 
     def add_api(self, swagger_file, base_path=None, arguments=None, auth_all_paths=None, swagger_json=None,
                 swagger_ui=None, swagger_path=None, swagger_url=None, validate_responses=False,
-                strict_validation=False, resolver=Resolver()):
+                strict_validation=False, resolver=Resolver(), resolver_error=None):
```

Comment:
It is not very clear that `resolver_error` is expected to be a HTTP status code. Would be nice to have a more friendly name here.